### ISSUE TITLE
Symbol ">" return an "invalidArgName" error

### ISSUE THREAD

**Issue by hug33k** [pvzig/SlackKit#61]
Hi,

I'm building a Slack Bot and when I try to send a text that contains the symbol `>`, I got the value `invalidArgName` in the `failure` closure of `client.webAPI.sendMessage`.

Example of code :
````swift
client.webAPI.sendMessage(myChannel, text: myMessage,
	username: myUsername, asUser: true, linkNames: true,
	iconEmoji: myEmoji, success: nil, failure: { (msg) in
		print("send Error:\(msg)")
})
````

Output :
`send Error:invalidArgName`

I don't have this issue with other symbols (`<`, `@`, ...) and when I do the request with cURL, it works.

````shell
$> curl -X GET -H "Cache-Control: no-cache" -H "Postman-Token: $TOKEN" "https://slack.com/api/chat.postMessage?token=$TOKEN&channel=$CHANNEL&text=>&parse=full&link_names=1&unfurl_links=true&unfurl_media=false&username=$NAME&as_user=true&icon_url=&icon_emoji=$EMOJI"

{"ok":true,"channel":"$CHANNEL","ts":"xxxxxxxxxx.xxxxxx","message":{"type":"message","user":"$USER","text":"&lt;","bot_id":"$BOTID","ts":"xxxxxxxxxx.xxxxxx"}}
````

Do you have an idea to resolve this?
Thanks!

**Comment by pvzig** [comment]
Hi Hugo,

This should be resolved in [3.1.7](https://github.com/pvzig/SlackKit/releases/tag/3.1.7). Let me know if you continue to have issues.

Peter

**Comment by hug33k** [comment]
Hi @pvzig,

Thanks, it works perfectly !
